locales/en-US/experiments.ftl

```diff
@@ -212,4 +212,4 @@ notesNewsupdatesNotesupdate1Content = We dramatically improved Notes by switchin
 sendNewsupdatesSendupdate1Title = Send with Safari and Edge!
 sendNewsupdatesSendupdate1Content = Send 1.1.0 just shipped with enhanced browser support and a few other improvements as well.
 sendNewsupdatesSendupdate2Title = Learning from Send
-sendNewsupdatesSendupdate2Content = We just conducted our first A/B test to learn more about Send’s user interface. Get the results.
+sendNewsupdatesSendupdate2Content = We just conducted our first A/B test to learn more about Send’s user interface. Get the results.
```

Comment:
This seems like an odd change. Not sure if something auto-trimmed the last blank line.